src/RTCCertificate.cpp

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (c) 2016, Andrew Gault and Nick Chadwick.
```

Comment:
Thanks Nick. Just to clarify cause I want to do the right/respectful thing here. Should I 1) append my name to only the new files, 2) to all files, 3) Put only my name on the new files I add? Obviously, I will increment the date on new files as well.